WordPress/src/test/java/org/wordpress/android/ui/sitecreation/verticals/NewSiteCreationVerticalsViewModelTest.kt

```diff
@@ -67,12 +71,17 @@ private val FAILED_HEADER_PROMPT_FETCHED = OnSegmentPromptFetched(
 
 private val FIRST_MODEL_ON_VERTICALS_FETCHED = OnVerticalsFetched(
         FIRST_MODEL_QUERY,
-        listOf(VerticalModel(FIRST_MODEL_NAME, FIRST_MODEL_ID)),
+        listOf(VerticalModel(FIRST_MODEL_NAME, FIRST_MODEL_ID, false)),
```

Comment:
Nitpick: It might be good to have the arguments here now that it's not obvious what `false` is referring to here. That goes for the other instances as well.

Reply:
👍